packages/docusaurus-plugin-content-docs/src/client/__tests__/docsClientUtils.test.ts

```diff
@@ -20,6 +21,18 @@ import type {
   GlobalDoc,
 } from '@docusaurus/plugin-content-docs/client';
 
+function createVersion(name: string, path: string, isLast: boolean) {
```

Comment:
Would be better to have 

```suggestion
function createVersion(partialVersion: Partial<VersionMetadata>) {
```

When multiple params have the same attribute, it's often better to have names, otherwise we rely on param position to know what it is and it makes tests less explicit